build.env

```diff
@@ -27,6 +27,11 @@ mkdir -p vthook
 export VTROOT="$PWD"
 export VTDATAROOT="${VTDATAROOT:-${VTROOT}/vtdataroot}"
 export PATH="$PWD/bin:$PATH"
+export PROTOC_VER=21.3
+export ZK_VER=${ZK_VERSION:-3.8.0}
```

Comment:
Why allow custom values only for ZK? I think in this case we want them to be fixed as we pull from our repo.

Reply:
`ZK_VER` had custom values in `bootstrap.sh`. So I did the same here.